### ISSUE TITLE
 pick window when vsplit open file not work!

### ISSUE THREAD

**Issue by vegawong** [kyazdani42/nvim-tree.lua#1233]
### Description

- Toggle nvim-tree to actived.
- Open A file
- Press `<C-w><C-w>` to switch `nvim-tree` window as actived.
- Selected B file and then press `<C-v>` to use vsplit open. 
- Now, the windows link that:  | `nvim-tree` | ` B file ` | ` A File ` |
- Switch `nvim-tree` window as actived.
- Selected C file and then press `<C-v>` to use vsplit open.
- Now it would has a prompt to pick window for A or B ? It would open the C file as a new window neither select A or select B.

### Neovim version

```markdown
NVIM v0.7.0
Build type: Release
LuaJIT 2.1.0-beta3
编译者 brew@BigSur

Features: +acl +iconv +tui
See ":help feature-compile"

     系统 vimrc 文件: "$VIM/sysinit.vim"
         $VIM 预设值: "/usr/local/Cellar/neovim/0.7.0/share/nvim"

Run :checkhealth for more info
```

### Operating system and version

macOS 11.5

### nvim-tree version

483f155

### Steps to reproduce

1. `nvim`
2.  `:NvimTreeToggle`
3. Select A file and press `<CR>`, open A file as a new window normally.
4. `<C-w><C-w>`, swith into nvimtree window.
5. Select B file and press `<C-v>`, open B file as a new split window.
6. Now, the windows link that:  | `nvim-tree` | ` B file ` | ` A File ` |
7. `<C-w><C-w><C-w><C-w>`, swith into nvimtree window.
8. Select C file and press `<C-v>`,  it would has a prompt to pick window for A or B ? It would open the C file as a new window neither select A or select B.

### Expected behavior

Pick window correctlly when open file with vsplit.

### Actual behavior

Open file as a new split window neither select window A o window B.

### Minimal config

```Lua
vim.cmd [[set runtimepath=$VIMRUNTIME]]
vim.cmd [[set packpath=/tmp/nvt-min/site]]
local package_root = "/tmp/nvt-min/site/pack"
local install_path = package_root .. "/packer/start/packer.nvim"
local function load_plugins()
  require("packer").startup {
    {
      "wbthomason/packer.nvim",
      "kyazdani42/nvim-tree.lua",
      "kyazdani42/nvim-web-devicons",
      -- ADD PLUGINS THAT ARE _NECESSARY_ FOR REPRODUCING THE ISSUE
    },
    config = {
      package_root = package_root,
      compile_path = install_path .. "/plugin/packer_compiled.lua",
      display = { non_interactive = true },
    },
  }
end
if vim.fn.isdirectory(install_path) == 0 then
  print "Installing nvim-tree and dependencies."
  vim.fn.system { "git", "clone", "--depth=1", "https://github.com/wbthomason/packer.nvim", install_path }
end
load_plugins()
require("packer").sync()
vim.cmd [[autocmd User PackerComplete ++once echo "Ready!" | lua setup()]]
vim.opt.termguicolors = true
vim.opt.cursorline = true

-- MODIFY NVIM-TREE SETTINGS THAT ARE _NECESSARY_ FOR REPRODUCING THE ISSUE
_G.setup = function()
  require("nvim-tree").setup {}
end
```

**Comment by alex-courtis** [comment]
Confirmed.

`window_picker.enable = false` results in desirable vsplits.

`open-file.fn()` could test `mode == "vsplit"` before calling the window picker.

**Comment by alex-courtis** [comment]
@vegawong please test this fix branch: #1236 

We will merge once you test.
